aten/src/ATen/native/quantized/TensorShape.cpp

```diff
@@ -0,0 +1,29 @@
+#include <ATen/ATen.h>
+#include <ATen/InferSize.h>
+#include <ATen/NativeFunctions.h>
+#include <ATen/TensorUtils.h>
+
+namespace at {
+namespace native {
+
+Tensor quantized_view(const Tensor& self, IntArrayRef size) {
```

Comment:
btw, is there anything qtensor specific here? or is it just that you refactored it out of TH? if so - maybe call it generically and also route this function for CPU/GPU to ATen instead of _th_view? (just in spirit of improving codebase)

Reply:
not really, I can remove _th_view

Reply:
then should I put this somewhere else?